### ISSUE TITLE
The --report-unused-disable-directives option in eslint doesn't work with this plugin

### ISSUE THREAD

**Issue by edg2s** [BenoitZugmeyer/eslint-plugin-html#111]
## Description

If eslint is run with the `--report-unused-disable-directives` option, it incorrectly reports all inline directives inside HTML script tags

## Setup configuration

* ESLint version: 5.12.0
* eslint-plugin-html version: 5.0.0
* NodeJS version: 6
* Operating System name and version: Ubuntu 18.04
* ESLint configuration (.eslintrc): any

## Aditional context

e.g.
```
<script>
  // eslint-disable-next-line no-eval
  eval( foo );
</script>
```

The `no-eval` directive is clearly required, but `eslint --report-unused-disable-directives` reports "Unused eslint-disable directive (no problems were reported from 'no-eval')".

The same code in a JS file report no such errors.

**Comment by BenoitZugmeyer** [comment]
Thank you for the report! Fixed in 5.0.2
